bin/pipeline_result_processor/prp/models/typing.py

```diff
@@ -0,0 +1,27 @@
+"""Typing related data models"""
+
+from typing import Dict, Optional
+
+from pydantic import Field
+
+from .base import RWModel
+
+
+class ResultMlstBase(RWModel):
+    """Base class for storing MLST-like typing results"""
+
+    alleles: Dict[str, int | None]
+
+
+class TypingResultMlst(ResultMlstBase):
+    """MLST results"""
+
+    scheme: str
+    sequence_type: int | None = Field(None, alias="sequenceType")
+
+
+class TypingResultCgMlst(ResultMlstBase):
```

Comment:
Consider Cgmlst or just straight up putting acronyms in uppercase

Reply:
I think `CgMLST` is visually easier to distinguish from `Mlst` and possibly `WgMlst` in the future